### PR TITLE
ioquake3: 2019-05-29 -> 2020-08-29

### PR DESCRIPTION
**Pull request by rvolosatovs**
###### Motivation for this change
Upstream update

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 96935` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>ioquake3</li>
    <li>quake3demo</li>
  </ul>
</details>

`gpl2` is a deprecated license, check if project uses `gpl2Plus` or `gpl2Only` and change `meta.license` accordingly.

File: pkgs/games/quake3/ioquake/default.nix [hunk 1]
```diff
@@ -4,13 +4,13 @@
...
   src = fetchFromGitHub {
     owner = "ioquake";
     repo = "ioq3";
-    rev = "350b8f9c7c88c002dccea4f0350f1919b86d3b4e";
-    sha256 = "17qkqi22f2fyh6bnfcf1zz2lycgv08d6aw52sf0hqw7r3qq86d08";
+    rev = "c0b15329e304013ac222a4cbba2458c407525553";
+    sha256 = "03s7icflms2hv41m0qgwk2rc8h8mhxhyy4xw3g398jlrl2pqd3mx";
   };
 
   nativeBuildInputs = [ which pkgconfig ];
```
Comment (SuperSandro2000) [inline pkgs/games/quake3/ioquake/default.nix:16]:
```suggestion
  nativeBuildInputs = [ which pkg-config ];
```

File: pkgs/games/quake3/ioquake/default.nix [hunk 2]
```diff
@@ -34,7 +34,7 @@ stdenv.mkDerivation {
   meta = with stdenv.lib; {
     homepage = "https://ioquake3.org/";
     description = "First person shooter engine based on the Quake 3: Arena and Quake 3: Team Arena";
-    license = licenses.gpl2;
+    license = licenses.gpl2Only;
```
Comment (rvolosatovs) [inline pkgs/games/quake3/ioquake/default.nix:37]: Wasn't sure which one to select, but this one seems the most applicable https://github.com/ioquake/ioq3/blob/05180e32dcfb9a4552e1b9652b56127248a9950c/COPYING.txt

**Comment by rvolosatovs** [comment]
@GrahamcOfBorg build ioquake3

**Review by SuperSandro2000 — APPROVED** [review]

**Comment by SuperSandro2000** [comment]
@ofborg eval

**Comment by SuperSandro2000** [comment]
Please change `pkgconfig` in line 1 to `pkg-config`.

**Comment by rvolosatovs** [comment]
@ofborg eval

**Comment by rvolosatovs** [comment]
@GrahamcOfBorg build ioquake3

**Pull request merged**